src/common/mfu_flist.c

```diff
@@ -50,6 +50,31 @@
 #include <sys/ioctl.h>
 #endif
 
+/* return a newly allocated walk_opts structure, set default values on its fields */
+mfu_walk_opts_t* mfu_walk_opts_new(void)
+{
+    mfu_walk_opts_t* opts = (mfu_walk_opts_t*)MFU_MALLOC(sizeof(mfu_walk_opts_t));
+
+    /* Don't set permissions on walk by default */
+    opts->dir_perms = 0;
+
+    /* Remove files in walk by default */
+    opts->remove = 1;
```

Comment:
Let's keep the defaults in opts_new() to be (dir_perms=0, remove=0, use_stat=1) since that's want most tools want.  Then in the drm tool, let's set opts->use_stat = 0 right after making the opts_new() call.

Reply:
@adammoody ok, updated.